content/fast/adaptive-serving-based-on-network-quality/index.md

```diff
@@ -0,0 +1,103 @@
+
+## Usage
+
+There are many ways you can use this network information to improve the user experience:
+
+* Switch between serving high-definition and low-definition content based on the user's network. 
+* Decide whether to preload resources. 
+* Defer uploads and downloads when users are on a slow connection. 
+* Enable offline mode if the network quality is not good enough to load the app and use the features.
+* Warn users that doing something (e.g. watching video) over cellular could cost them money.
+* Use it in your analytics to gather data on your users’ network quality.
+
+Many applications are already doing something similar. For example, YouTube, Netflix and most other video (or video calling) services automatically adjust the resolution during streaming. When Gmail is loading, it provides users with a link to "load basic HTML (for slow connections)" - a simplified (yet functional) version of Gmail. 
+
+<img class="screenshot" src="./loading_gmail_slow_connection.png" alt="A link to load basic HTML version of Gmail when users are on slow connections">
+
+With the Network Information API you can now implement these functionalities in your own applications without much effort.
+
+## How it works
+
+The **navigator.connection** object contains information about a client’s connection. Its properties are:
+
+1. **downlink**: the bandwidth estimate in megabits per second
+2. **effectiveType**: the effective type of the connection, with possible values **‘slow-2g’**,**‘2g’**, **‘3g’**, or **‘4g’**
+3. **onchange:** an event handler that fires when connection information changes
+4. **rtt**: the estimated round-trip latency of the connection in milliseconds
+5. **saveData:** boolean that defines whether the user has requested a reduced data usage mode 
+
+Here’s what this looks like when you run it in the browser’s console:
+
+<img class="screenshot" src="./network_information_in_console.jpg" alt="Chrome DevTools console displaying the values of navigator.connection object's properties">
+
+It might seem odd that the **effectiveType **only goes up to ‘**4g’**, but that segment actually includes all connections that are **‘4g’ and higher.** The API determines the effective connection type based on the combination of **round-trip time (RTT) and downlink speed**. For example, if you have a 10 Mbps downlink combined with 1000 ms latency, the **effectiveType **will be ‘3g’ due to the latency. 
+
+The **effectiveType** values are also available via [Client Hints](https://www.chromestatus.com/features/5407907378102272) and allow you to communicate the browser's connection type to servers.
+
+<table>
```

Comment:
I'm not sure I see a lot of value in this table, explaining effectiveType usage. It's not being used here, other than as a descriptor. And it is taking up a lot of space. Would you be open to removing it?

Reply:
Removed the table from the doc and included a link to the spec.